scripts/hatch_build.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import annotations
+
+import json
+import os
+import sys
+import typing as t
+
+from pathlib import Path
+
+from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+
+BASE_DIR = Path(__file__).parents[1]
+GREEN, RED, RESET = "\033[92m", "\033[91m", "\033[0m"
+
+
+def build_models():
```

Comment:
I feel like build hooks should be at the root of the repository.

Reply:
Yeah, I was not sure about this one. I like the idea of having as clean (as possible) root, mainly with directories and config files.  

I can move it into the root directory if you feel strongly about it.

Reply:
I feel strongly about it :)